### DISCUSSION TITLE
Feature Request: Assignees

### DISCUSSION THREAD

**Discussion by melloware** [community/community#2897] [Mobile] [answered]
When going to assign an issue to someone only members of the project are in the list. 

On the GitHub full website the assignee can be any of the project members OR any of the members who have created or commented on the ticket.

**Comment by candyho** [comment, marked as answer]
Hey thanks for your feedback! Have you tried to search for the user you would like to assign in the assignees view and see if their names would appear? It should work.

**Reply by melloware** [reply]
No the search does not work it comes up with "no results found" when I search the author of the Issue I am looking at.
